### ISSUE TITLE
Fish highlights only apply if the Nix attribute sets are nested a certain way

### ISSUE THREAD

**Issue by mrjones2014** [calops/hmts.nvim#13]
For example, to get Fish highlighting, this works:

```nix
programs.fish = {
  functions = {
    my-function = ''
      echo "some shell script"
    '';
  };
};

# this structure also works
programs.fish.functions = {
  my-function = ''
    echo "some shell script"
  '';
};
```

However this does _not_ work:

```nix
programs.fish = {
  functions.my-function = ''
    echo "some shell script"
  '';
};
```

**Comment by calops** [comment]
This is due to a mistake in how the `hmts-path` directive was implemented. Shouldn't be too hard to solve, expect a fix this weekend.

**Comment by calops** [comment]
This should be fixed in #14, can you confirm?

**Comment by mrjones2014** [comment]
It works! Thanks for the quick turnaround on these. This is a really awesome plugin!
